gr-iio/lib/CMakeLists.txt

```diff
@@ -0,0 +1,73 @@
+# Copyright 2011,2012 Free Software Foundation, Inc.
+#
+# This file is part of GNU Radio
+#
+# GNU Radio is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3, or (at your option)
+# any later version.
+#
+# GNU Radio is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with GNU Radio; see the file COPYING.  If not, write to
+# the Free Software Foundation, Inc., 51 Franklin Street,
+# Boston, MA 02110-1301, USA.
+
+########################################################################
+# Setup library
+########################################################################
+add_library(gnuradio-iio
+  device_source_impl.cc
+  device_sink_impl.cc
+  attr_source_impl.cc
+  attr_sink_impl.cc
+  attr_updater_impl.cc
+  dds_control.cc
+)
+
+target_link_libraries(gnuradio-iio PUBLIC
+  gnuradio-runtime
```

Comment:
Since this new `gr-iio` submodule depends on GR's `analog` and ` blocks` in its top-level CMakeLists.txt, shouldn't those libraries be included here for linking? I think this is what @carlesfernandez was meaning about https://github.com/analogdevicesinc/gnuradio/pull/2 ...

Reply:
Integrated the PR

Reply:
thx!